### PR TITLE
Updates contributors

### PR DESCRIPTION
**Pull request by ChaosExAnima**
Updates contributor list.

#### Changes proposed in this Pull Request:

* Updates contributors to be up to date.

#### Proposed changelog entry for your changes:

* N/A

### CODE REVIEW

**Review by jeherve — CHANGES_REQUESTED** [review]
Cool!

You're part of the team, so you could add yourself in the first list above where you've added yourself. We can also take that opportunity to remove folks not in the team anymore.

I also think you could add yourself to the contributor list in the `readme.txt` file as well, so you'll appear as a plugin author on WordPress.org.

**Comment by ChaosExAnima** [comment]
I updated `readme.txt` as well! I'm not sure who is a stale contributor, however?

File: readme.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ Jetpack is licensed under [GNU General Public License v2 (or later)](./LICENSE.t
 
 ## Team
 
-Jetpack is developed and maintained by various teams at Automattic. The Jetpack Crew responsible for coordinating everything is comprised of @brbrr, @briancolinger, @dereksmart, @eliorivero, @jeherve, @kraftbj, @mdawaffe, @oskosk, and @zinigor.
+Jetpack is developed and maintained by various teams at Automattic. The Jetpack Crew responsible for coordinating everything is comprised of @brbrr, @dereksmart, @chaosexanima, @jeherve, @kbrownkd, @kraftbj, @mdawaffe, and @zinigor.
```
Comment (jeherve) [inline readme.md]:
```suggestion
Jetpack is developed and maintained by various teams at Automattic. The Jetpack Crew responsible for coordinating everything is comprised of @brbrr, @dereksmart, @chaosexanima, @jeherve, @kbrown9, @kraftbj, @mdawaffe, and @zinigor.
```

File: readme.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+Contributors: automattic, adamkheckler, aduth, akirk, allendav, alternatekev, andy, annezazu, apeatling, azaozz, batmoo, barry, beaulebens, blobaugh, cainm, cena, cfinke, chaselivingston, chellycat, clickysteve, csonnek, danielbachhuber, davoraltman, daniloercoli, designsimply, dllh, drawmyface, dsmart, dzver, ebinnion, egregor, eliorivero, enej, eoigal, erania-pinnera, ethitter, gcorne, georgestephanis, gibrown, goldsounds, hew, hugobaeta, hypertextranch, iammattthomas, iandunn, jblz, jasmussen, jeffgolenski, jeherve, jenhooks, jenia, jessefriedman, jgs, jkudish, jmdodd, joanrho, johnjamesjacoby, jshreve, keoshi, koke, kraftbj, lancewillett, lschuyler, macmanx, martinremy, matt, matveb, mattwiebe, maverick3x6, mcsf, mdawaffe, MichaelArestad, migueluy, mikeyarce, mkaz, nancythanki, nickmomrik, obenland, oskosk, pento, professor44, rachelsquirrel, rdcoll, ryancowles, richardmuscat, richardmtl, roccotripaldi, samhotchkiss, scarstocea, sdquirk, stephdau, tmoorewp, tyxla, Viper007Bond, westi, yoavf, zinigor
```
Comment (jeherve) [inline readme.txt]:
```suggestion
Contributors: automattic, adamkheckler, aduth, akirk, allendav, alternatekev, andy, annezazu, apeatling, azaozz, batmoo, barry, beaulebens, blobaugh, cainm, cena, cfinke, chaselivingston, chellycat, clickysteve, csonnek, danielbachhuber, davoraltman, daniloercoli, designsimply, dllh, drawmyface, dsmart, dzver, ebinnion, egregor, eliorivero, enej, eoigal, erania-pinnera, ethitter, gcorne, georgestephanis, gibrown, goldsounds, hew, hugobaeta, hypertextranch, iammattthomas, iandunn, jblz, jasmussen, jeffgolenski, jeherve, jenhooks, jenia, jessefriedman, jgs, jkudish, jmdodd, joanrho, johnjamesjacoby, jshreve, kbrownkd, keoshi, koke, kraftbj, lancewillett, lschuyler, macmanx, martinremy, matt, matveb, mattwiebe, maverick3x6, mcsf, mdawaffe, MichaelArestad, migueluy, mikeyarce, mkaz, nancythanki, nickmomrik, obenland, oskosk, pento, professor44, rachelsquirrel, rdcoll, ryancowles, richardmuscat, richardmtl, roccotripaldi, samhotchkiss, scarstocea, sdquirk, stephdau, tmoorewp, tyxla, Viper007Bond, westi, yoavf, zinigor
```

**Review by jeherve — CHANGES_REQUESTED** [review]
Just a couple more changes.

**Review by jeherve — APPROVED** [review]

**Comment by jeherve** [comment]
Cherry-picked to `branch-7.9` in 29feed6349de956300d428498950b23fd58d982d

**Pull request merged**